docs/source/models/other_models/facility_choice/index.rst

```diff
@@ -445,6 +453,24 @@ more details on assigning ANC status; see the `Causal conditional
 probabilities for in-facility delivery`_ section below for an explicit
 description of how to assign IFD status.
 
+.. note::
+
+  The facility choice causal optimization model has not yet been updated
+  to make use of all four ANC attendance categories or the corresponding
+  additional detail for ultrasound timing. Accordingly, the
+  :ref:`AI-ultrasound module
+  <2024_vivarium_mncnh_portfolio_ai_ultrasound_module>` currently groups
+  the last three ANC categories together, effectively making ANC
+  attendance a dichotomous variable with categories ordered "no ANC" <
+  "some ANC".
+
+  In a future version of the model, we plan to use the more detailed ANC
+  attendance information to determine whether simulants get a standard
+  ultrasound in the 1st trimester or in later pregnancy, which affects
+  the accuracy of GA estimation. Making these changes will require
+  updating the facility choice causal optimization code and the final
```

Comment:
It would be cool to link to that code here.  Is it in one of the notebooks here? https://github.com/ihmeuw/vivarium_research_mncnh_portfolio/tree/main/facility_choice

Reply:
The code is in the Python modules there (which I probably want to move somewhere else in order to make them pip installable). The folder is linked below, but I can add another link here.